examples/pos-mainchain/config/default/genesis_assets_with_preset.json

```diff
@@ -0,0 +1,3074 @@
+					{
+						"storeKey": "04000000",
+						"storeValue": {
+							"messageFeeTokenID": "0400000000000000",
+							"outbox": {
+								"appendPath": [],
+								"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+								"size": 0
+							},
+							"inbox": {
+								"appendPath": [],
+								"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+								"size": 0
+							},
+							"partnerChainOutboxRoot": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					},
+					{
+						"storeKey": "04000001",
+						"storeValue": {
+							"messageFeeTokenID": "0400000000000000",
+							"outbox": {
+								"appendPath": ["d3eb21361c783b37106d1089243f047ae53b4d3ddb95256ad0120ec9ca40b3a7"],
+								"root": "d3eb21361c783b37106d1089243f047ae53b4d3ddb95256ad0120ec9ca40b3a7",
+								"size": 1
+							},
+							"inbox": {
+								"appendPath": [],
+								"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+								"size": 0
+							},
+							"partnerChainOutboxRoot": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					},
+					{
+						"storeKey": "04000002",
+						"storeValue": {
+							"messageFeeTokenID": "0400000000000000",
+							"outbox": {
+								"appendPath": ["2381eae35fbef3018bfe5ef929e33c04548ef6a6f50d0e9bc6981e3a2eb94b2c"],
+								"root": "2381eae35fbef3018bfe5ef929e33c04548ef6a6f50d0e9bc6981e3a2eb94b2c",
+								"size": 1
+							},
+							"inbox": {
+								"appendPath": [],
+								"root": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855",
+								"size": 0
+							},
+							"partnerChainOutboxRoot": "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855"
+						}
+					}
+				],
+				"chainValidatorsSubstore": [
+					{
+						"storeKey": "04000000",
+						"storeValue": {
+							"activeValidators": [
+								{
+									"blsKey": "92f020ce5e37befb86493a82686b0eedddb264350b0873cf1eeaa1fefe39d938f05f272452c1ef5e6ceb4d9b23687e31",
+									"bftWeight": "2"
+								}
+							],
+							"certificateThreshold": "2"
+						}
+					},
+					{
+						"storeKey": "04000001",
+						"storeValue": {
+							"activeValidators": [
+								{
+									"blsKey": "92f020ce5e37befb86493a82686b0eedddb264350b0873cf1eeaa1fefe39d938f05f272452c1ef5e6ceb4d9b23687e31",
+									"bftWeight": "2"
+								}
+							],
+							"certificateThreshold": "2"
+						}
+					},
+					{
+						"storeKey": "04000002",
+						"storeValue": {
+							"activeValidators": [
+								{
+									"blsKey": "92f020ce5e37befb86493a82686b0eedddb264350b0873cf1eeaa1fefe39d938f05f272452c1ef5e6ceb4d9b23687e31",
+									"bftWeight": "2"
+								}
+							],
+							"certificateThreshold": "2"
+						}
+					}
+				],
+				"ownChainDataSubstore": [
+					{
+						"storeKey": "",
+						"storeValue": {
+							"name": "lisk_mainchain",
+							"chainID": "04000000",
+							"nonce": 0
```

Comment:
nonce cannot be 0 if another chain is registered (since we at least have sent the registration message)